canonical-web-and-design/jaas-dashboard
# Create a basic panel to hold the Actions content

## Done

- Create a basic panel to hold the Actions content
- Create a `PanelHeader` component and update the `RegisterController` panel with it. It's the only other panel using the new panel system.
- Move the `RegisterController` panel into the `panels/` folder.
- Added a temporary configuration value for showing a button to render the actions panel.

## QA

- Pull code
- `yarn start`
- Open `config.js` and change `showActionsPanel` to `true`.
- View an application and clicking the "Actions" button should open a panel that can be closed.
- View the "Register Controller" panel, it should have a functioning close button as well.

## Details

Fixes https://github.com/canonical-web-and-design/juju-squad/issues/1661
